cmd/collector/app/handler/grpc_handler.go

```diff
@@ -49,6 +51,9 @@ func (g *GRPCHandler) PostSpans(ctx context.Context, r *api_v2.PostSpansRequest)
 		SpanFormat:       processor.ProtoSpanFormat,
 	})
 	if err != nil {
+		if err == processor.ErrBusy {
+			return nil, status.Errorf(codes.ResourceExhausted, err.Error())
```

Comment:
@yurishkuro it wasn't clear to me what error should we use here https://grpc.io/docs/guides/error/

Reply:
I think `GRPC_STATUS_RESOURCE_EXHAUSTED`